src/python/tests/core/crash_analysis/stack_parsing/stack_analyzer_test.py

```diff
@@ -2435,3 +2435,174 @@ def test_log_fatal_google(self):
     self._validate_get_crash_data(data, expected_type, expected_address,
                                   expected_state, expected_stacktrace,
                                   expected_security_flag)
+
+  def test_golang_asan_panic(self):
+    """Test golang stacktrace with panic and ASan."""
+    data = self._read_test_data('golang_asan_panic.txt')
+    expected_type = 'ASSERT'
+    expected_address = ''
+    expected_state = (
+        'asn1: string not valid UTF-8\nasn1.Fuzz\nruntime.raise\n')
```

Comment:
need to add runtime.raise in ignore list, so that it does not appear here.

Reply:
Ah, this actually comes from the libFuzzer stacktrace, we may want to do this instead: https://github.com/google/oss-fuzz/pull/2188#issuecomment-521815547

But I'm fine with suppressing in too, will add.

Reply:
yes need to suppress for better deduplication.

Reply:
Done